posthog/api/feature_flag.py

```diff
@@ -35,6 +38,15 @@ class Meta:
             "rollout_percentage",
         ]
 
+    def __init__(self, instance: Optional[Any] = None, data: Any = empty, **kwargs: Any):
```

Comment:
This is suspect, I don't think we use this pattern anywhere else.

Is there a reason we need to override init over e.g. https://stackoverflow.com/questions/46173838/validate-specific-field-in-drf-serializer?

Reply:
I've done this before in other projects and it works fine, but I opted for what you suggested.